pkg/cgroups/cgroups_windows.go

```diff
@@ -6,12 +6,16 @@ import (
 	"errors"
 )
 
+var (
+	notSupportedWinErr = errors.New("not supported on windows")
```

Comment:
Hey, little nit, I'd like to enable the linter for that on Windows but that name should be something like:
```suggestion
	ErrWindowsNotSupported = errors.New("not supported on Windows")
```
or
```suggestion
	ErrNotSupportedOnWindows = errors.New("not supported on Windows")
```
error name should start with "Err" https://go.dev/wiki/Errors#naming

Reply:
Do you suggest we change the name in existing code too ?

Reply:
Apply the naming convention on other already existing error in the code? yeah we can but we can do it in another PR :)

Reply:
if you reuse this elsewhere, maybe it's best to export it and put this in an appropriate package

Reply:
I have added an error in constnats package and used it globally. Thanks for the suggestion and guidance.